shell/platform/fuchsia/runtime/dart/utils/build_info_in.cc

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "runtime/dart/utils/build_info.h"
+
+namespace dart_utils {
+
+const char* BuildInfo::FlutterEngineGitRevision() {
+  return "{{FLUTTER_ENGINE_GIT_REVISION}}";
+}
+
+const char* BuildInfo::DartSdkGitRevision() {
+  return "{{DART_SDK_GIT_REVISION}}";
+}
+
+const char* BuildInfo::DartSdkSemanticVersion() {
+  return "{{DART_SDK_SEMANTIC_VERSION}}";
+}
+
+const char* BuildInfo::BuildTimestamp() {
+  return "{{BUILD_TIMESTAMP}}";
+}
+
+void BuildInfo::Dump() {
```

Comment:
Can you pass in the node here? If you pass the node then we do not have to worry about the lifecycle here, rather it just writes the values and you can hold on to the build_info node in main which will remove all statics.

Reply:
Actually, you may ned to pass in the inspector instead of the node here so that you can use the CreateString() method that holds on to the reference for you.

Reply:
okay. I'll incorporate Naud's change of exposing the component's inspector and access it that way.